keyboards/skyloong/gk980/bl/bl.c

```diff
@@ -0,0 +1,87 @@
+//  Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+bool dis_breath = 0;
+uint8_t breath_level = 0;
+
+void suspend_power_down_kb() {
+    gpio_write_pin_low(MAC_PIN);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    switch (get_highest_layer(layer_state)) {
```

Comment:
I think you mean default_layer_state here. 

Also, the default_layer_state_set_user should be moved here, and to the kb function.

Reply:
OK, I do it.